postgres/pixl-db_init.sh

```diff
@@ -16,7 +16,7 @@
 set -euxo pipefail
 
 # Create the EHR schema and associated tables
-columns_and_types="mrn text, accession_number text, age integer, sex text, ethnicity text, height real, weight real, gcs integer, xray_report text"
+columns_and_types="mrn text, accession_number text, image_identifier text, procedure_occurrence_id integer, age integer, sex text, ethnicity text, height real, weight real, gcs integer, xray_report text, project_name text, extract_datetime timestamp"
```

Comment:
Is it worth tracking with timezone?

Reply:
Do we have that information? If so, then yes.

Reply:
From the example logfile:
```
"datetime":"2023-12-07 14:08:58",
```
This doesn't specify a timezone or an offset, although if we're sure it's UTC we could add that information ourselves.
However, do we use it as anything other than a fairly opaque identifier for an extract? If it's not a product output itself and we're not doing any actual calculations on it, I don't think we need to care about the timezone.

Reply:
yeah that's fair, in that case happy with keeping it as-is, as we don't know or particularly care about the timezone other than an identifier